tests/assets/compat_arg_complete_click7_8.py

```diff
@@ -0,0 +1,22 @@
+import click
+import typer
+
+app = typer.Typer()
+
+
+def shell_complete(ctx: click.Context, param: click.Parameter, incomplete: str):
```

Comment:
It would also be nice to test without `ctx` and `param`. In many cases I just use something like `shell_complete(incomplete: str)`. I think this requires the use of `get_param_completion`, see my other comment.